lib/views/custom_widgets/dialogs/photo_booth_dialog.dart

```diff
@@ -29,7 +29,7 @@ class PhotoBoothDialog extends StatelessWidget {
       margin: const EdgeInsets.all(32.0),
       padding: const EdgeInsets.symmetric(horizontal: 32.0, vertical: 24.0),
       decoration: ShapeDecoration(
-        color: const Color(0xFFFFFFFF).withOpacity(0.90),
+        color: const Color(0xFFFFFFFF).withValues(alpha: 0.1),
```

Comment:
This is incorrect I think! The dialog is almost transparent now.

Reply:
It seems... that... I might have been confusing stuff here...? Alpha really is the same as opacity... not the opposite.